src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -80,6 +88,18 @@ const TablesUserList: React.FunctionComponent<PropsType> = ({
   if (!isRecentVenueUsersLoaded) return <>Loading...</>;
 
   const tables: Table[] = customTables || defaultTables;
+
+  const emptyTables = tables.filter((table) => {
+    if (!table.capacity) return false;
+
+    const usersSeatedAtTable = recentVenueUsers.filter(
+      (u) => u.data?.[venueName].table === table.reference
```

Comment:
`u.data` will be `undefined` if the user has never entered a table in the venue. This will blow up if you are entering for the first time in the venue and this component renderes.

Reply:
Nice catch! Really thank you for your attention. fixed in 628728c978f8946518261612288a3324163ca176